R/objects.R

```diff
@@ -1525,6 +1585,66 @@ as.Graph.matrix <- function(x, ...) {
   return(as.Graph.Matrix(x = as(object = x, Class = 'Matrix')))
 }
 
+#' @param counts name of the CellDataSet assay to store as \code{counts}
+#' @param data name of the CellDataSet assay to slot as \code{data}
+#'
+#' @rdname as.Seurat
+#' @export
+#' @method as.Seurat CellDataSet
+#'
+as.Seurat.CellDataSet <- function(x, counts = "counts", ...) {
+  if (!PackageCheck('monocle', error = FALSE)) {
+    stop("Please install monocle from Bioconductor before converting to a CellDataSet object")
+  }
+  counts <- tryCatch(
+    expr = exprs(x),
```

Comment:
`exprs` was not imported. Remember to import all functions used with Roxygen comments in the documentation

```R
#' @importFrom <packageName> exprs
```